cliquet/tests/support.py

```diff
@@ -34,8 +34,8 @@ def __init__(self, *args, **kwargs):
         self.headers = {}
         self.errors = cornice_errors.Errors(request=self)
         self.authenticated_userid = 'bob'
-        self.authn_type = 'basicauth'
-        self.prefixed_userid = 'basicauth:bob'
+        self.authn_type = 'basic'
+        self.prefixed_userid = 'basic:bob'
```

Comment:
I would have left ``basicauth`` since this is the actual value for ``BasicAuthAuthenticationPolicy``